migrations/20170906161500-healthcheck_interval.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+    up: function (queryInterface, DataTypes) {
+        let changes = [];
+
+        queryInterface.describeTable('Ports').then(attributes => {
+          if (!attributes.hasOwnProperty('healthcheck_interval')) {
+            // healthcheck_interval
+            changes.push(
+                queryInterface.addColumn(
+                    'Ports',
+                    'healthcheck_interval',
+                    {
+                        type: DataTypes.INTEGER,
+                        field: "healthcheck_interval",
+                        validate: {
+                            min: 1,
+                            max: 3600
+                        },
+                        defaultValue: 10,
+                        get() {
+                            return this.getDataValue('healthcheck_interval') || this.getDataValue('value');
+                        }
+                    }
+                )
+            );
+          }
+        });
+
+        return Promise.all(changes);
+    },
+
+    down: function (queryInterface, DataTypes) {
+        let changes = [];
+
+        // healthcheck_interval
+        changes.push(
```

Comment:
while I know we said this does not matter when talking in person, it seems very non intuitive to potential new users of this api, that this code is useless. Could we just return some promise that does nothing?

Or at the very least add a comment that says this is useless.

Reply:
Good feedback, will just return an empty array with the promise